frontend/src/store/modules/participant.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+
+export default {
+  namespaced: true,
+  state: {
+    allParticipants: [],
+    error: null
+  },
+  getters: {
+    getParticipants(state) {
+      return state.allParticipants;
+    },
+    getError(state) {
+      return state.error;
+    }
+  },
+  mutations: {
+    raiseError(state, error_msg) {
+      state.error = error_msg;
```

Comment:
This is probably unnecessary but defaults error messages are rendered in polish. Although I don't think it's necessary to append them with our own messages.